OrbitQt/orbitmainwindow.ui

```diff
@@ -228,7 +238,7 @@
      <x>0</x>
      <y>0</y>
      <width>976</width>
-     <height>21</height>
+     <height>20</height>
```

Comment:
Is this an intended change?

Reply:
no, it is not

Reply:
Then let's change it back to 21.

Reply:
What I meant is that the code was automatically changed in Qt designer and I have not touched that file manually.

Reply:
Ah, I see. Let's bring this up in the standup tomorrow - someone might have bumped into this before.

Reply:
Sure. Thanks.